db/storage_options.hh

```diff
@@ -27,7 +27,7 @@
 
 struct storage_options {
     enum class storage_type {
-        NATIVE, S3,
+        NATIVE, S3, LOCAL_SHARED,
```

Comment:
Shared doesn't have to be local.

Reply:
And S3 is shared too! I think the distinguishing features is that it is a shared filesystem (whether local or networked, such as with NFS).

Reply:
What I meant to express here is that the mount point is local, and this option is configured by supplying a local fs path as the bucket. I'm not emotionally attached to this name though, so maybe I'll switch to SHARED_FS or something like that